routes/apiRoutes.js

```diff
@@ -8,6 +8,43 @@ module.exports = function(app) {
     });
   });
 
+  // Get all inventory
```

Comment:
This and the other routes would make a lot of data we'd want to keep private public by taking all the information from each table. We should talk on Saturday about what routes we'll need.

Reply:
Sounds good